scrapy/pipelines/files.py

```diff
@@ -213,19 +214,24 @@ class FilesPipeline(MediaPipeline):
     """
 
     MEDIA_NAME = "file"
-    EXPIRES = 90
     STORE_SCHEMES = {
         '': FSFilesStore,
         'file': FSFilesStore,
         's3': S3FilesStore,
     }
-    DEFAULT_FILES_URLS_FIELD = 'file_urls'
-    DEFAULT_FILES_RESULT_FIELD = 'files'
 
-    def __init__(self, store_uri, download_func=None):
+    def __init__(self, store_uri, settings=None, download_func=None):
```

Comment:
Could you please move `settings` argument to the end to make it backwards compatible for users who user positional arguments to create FilesPipeline instance?